vertical-pod-autoscaler/pkg/admission-controller/Makefile

```diff
@@ -32,6 +34,19 @@ ifndef TAG
 endif
 	docker build --pull -t ${REGISTRY}/${FULL_COMPONENT}:${TAG} .
 
+docker-build-and-publish-multiarch:
+ifndef INTERNAL_REGISTRY
+	ERR = $(error INTERNAL_REGISTRY is undefined)
+	$(ERR)
+endif
+ifndef TAG
+	ERR = $(error TAG is undefined)
+	$(ERR)
+endif
+	docker buildx create --config /etc/cdp-buildkitd.toml --driver-opt network=host --use
+	docker buildx build --rm -t ${INTERNAL_REGISTRY}/${FULL_COMPONENT}:${TAG} --platform linux/amd64,linux/arm64 --push .
+	cdp-promote-image ${INTERNAL_REGISTRY}/${FULL_COMPONENT}:${TAG}
```

Comment:
I would propose to move all this to the `delivery.yaml` to avoid patching the upstream `Makefile` too much. The less we depend on changes to the upstream code, the easier it would potentially be to adopt upstream in the future.